query-engine/driver-adapters/js/adapter-neon/package.json

```diff
@@ -21,7 +21,8 @@
     "@prisma/driver-adapter-utils": "workspace:*"
   },
   "devDependencies": {
-    "@neondatabase/serverless": "^0.6.0"
+    "@neondatabase/serverless": "^0.6.0",
+    "postgres-array": "^3.0.2"
```

Comment:
I think this actually needs to be a real dependency in `dependencies`.
 
I don't think we would bundle this, so it has to be installed.
Or _do_ we bundle so that the devDependencies are included?